### ISSUE TITLE
Could not verify patch for HandleRageGain()::NoHeavyRageGain

### ISSUE THREAD

**Issue by naydef** [nosoop/SM-TFCustomAttributeStarterPack#89]
**Initial troubleshooting checklist**
- [X] The attribute is loaded, or this bug report is about a plugin load failure.  (Check `sm plugins list` on the server.)
- [ ] The attribute is the most recent revision.

**Describe the bug**
```
L 04/21/2024 - 09:40:53: [SM] Exception reported: Could not verify patch for HandleRageGain()::NoHeavyRageGain
L 04/21/2024 - 09:40:53: [SM] Blaming: Gameplay/Weapons/Attributes/generate_rage_on_dmg_patch.smx
L 04/21/2024 - 09:40:53: [SM] Call stack trace:
L 04/21/2024 - 09:40:53: [SM]   [0] SetFailState
L 04/21/2024 - 09:40:53: [SM]   [1] Line 38, Gameplay/Weapons/Attributes/generate_rage_on_dmg_patch.sp::OnPluginStart
L 04/21/2024 - 09:40:53: [SM] Unable to load plugin "Gameplay/Weapons/Attributes/generate_rage_on_dmg_patch.smx": Error detected in plugin startup (see error logs)
```

**Server environment (please complete the following information):**
 - Operating System: Rocky Linux
 - SourceMod version: SourceMod Version: 1.11.0.6958

**Comment by higps** [comment]
I'm having the same issue

**Comment by naydef** [comment]
I made new gamedata for Linux (only):
Here's the diff:
```diff
@@ -849,9 +849,9 @@
 				"signature"			"CTFPlayer::ApplyPushFromDamage()"
 				"linux"
 				{
-					"offset"		"684h"
-					"verify"		"\x74"
-					"patch"			"\x71"
+					"offset"		"35Eh"
+					"verify"		"\x0F\x84\x24\x0C\x00\x00"
+					"patch"			"\x90\x90\x90\x90\x90\x90"
 				}
 				"windows"
 				{
@@ -882,9 +882,9 @@
 				"signature"			"HandleRageGain()"
 				"linux"
 				{
-					"offset"		"480h"
-					"verify"		"\x0F\x84"
-					"patch"			"\x0F\x81"
+					"offset"		"6Ch"
+					"verify"		"\x74\x62"
+					"patch"			"\x90\x90"
 				}
 				"windows"
 				{
@@ -899,7 +899,7 @@
 				"signature"			"CTFWeaponBase::ApplyOnHitAttributes()"
 				"linux"
 				{
-					"offset"		"8E2h"
+					"offset"		"5C5h"
 					"verify"		"\x74"
 					"patch"			"\xEB"
 				}
@@ -1009,7 +1009,7 @@
 			{
 				// signature with ApplyOnDamageAliveModifyRules starting with MULSS operation
 				"library"	"server"
-				"linux"		"\xF3\x0F\x59\x2A\x2A\x2A\x2A\x2A\xF3\x0F\x11\x2A\x2A\x2A\x2A\x2A\xE9\x2A\x2A\x2A\x2A\x83\x3D\x2A\x2A\x2A\x2A\x00"
+				"linux"		"\xF3\x0F\x59\x2A\x2A\x2A\x2A\x2A\xF3\x0F\x11\x2A\x2A\x2A\x2A\x2A\xE9\x2A\x2A\x2A\x2A\x90\x31\xC0"
 			}
 			"CTFGameRules::RadiusDamage()"
 			{
```

**Comment by naydef** [comment]
For custom_lunchbox_effects gamedata diff:
```diff
 				"signature"			"CTFPlayer::DoTauntAttack()"
 				"linux"
 				{
-					"offset"		"91"
+					"offset"		"88"
 				}
 				"windows"
 				{
```

**Comment by nosoop** [comment]
Closed with #91.  Thanks!
Couple of other issues left that will be tracked under #90.
